src/main/resources/application.properties

```diff
@@ -139,3 +139,4 @@ feature.toggle.contested_print_draft_order_not_approved=${FEATURE_CONTESTED_PRIN
 feature.toggle.contested_print_general_order=${FEATURE_CONTESTED_PRINT_GENERAL_ORDER:false}
 feature.toggle.share_a_case=${FEATURE_SHARE_A_CASE:false}
 feature.toggle.send_to_frc=${FEATURE_SEND_TO_FRC:false}
+feature.toggle.offline_notifications=${FEATURE_OFFLINE_NOTIFICATIONS:true}
```

Comment:
This should be false by default I think.

Reply:
yeah you can set it to true in the Chart rather than defaulting here